csun-metalab/faculty-v3
# Include Name Pronuncation Audio From Media Webservice

This PR makes an update to the profile-banner section (of each faculty member's profile-page) to include name-pronunciation-audio from the media webservice.

To test:

- look at Steve's profile page to see the newly included audio icon next to his name. Click the icon to hear Steve's name. Play around with it and make sure there are no errors or issues.
- look at Deborah Chen's profile page (or most other faculty members) - she has not uploaded an audio file for her name, so there is no audio icon next to her name. Make sure there are no errors or issues.
